h2o-r/h2o-package/R/automl.R

```diff
@@ -412,7 +412,7 @@ h2o.predict.H2OAutoML <- function(object, newdata, ...) {
   is_progress <- isTRUE(as.logical(.h2o.is_progress()))
   if (show_progress) h2o.show_progress() else h2o.no_progress()
   frame <- tryCatch(
-    as.h2o(table, destination_frame=destination_frame),
+    suppressWarnings(as.h2o(table, destination_frame=destination_frame)),
```

Comment:
Are the warning suppressed because of data.table?

IMHO we should address it better:
- in this context the tables will be slow, it doesn't even make sense to attempt to use data.table
- as.h2o could have an option that would disable d.t and or/ d.t would be triggered only for frames of certain sizes

Reply:
this suppresses the warning of data.table being installed without bit64, which means we do not use it

Reply:
yes, that is what I thought - my point is we should give users a mechanism to disable use of data.table (in as.h2o)

we know the data is small enough and DT won't provide any benefit

another point of view is that we don't even need to attempt to use DT when the data is this small

Reply:
Yes, it's because of this message:
```
warning("data.table cannot be used without R package bit64 version 0.9.7 or higher.  Please upgrade to take advangage of data.table speedups.")
```

Reply:
we do

```use=getOption("h2o.use.data.table", TRUE)[package=="data.table"]```

Reply:
> in as.h2o - not just using a global flag

not in as.h2o

Reply:
The point I am trying to make is that we are suppressing a warning from something we don't need at all in this case.

And possibly other (unrelated) warnings. Instead of that we could just disable data.table in a specific call to `as.h2o`. Does that make sense?

Reply:
data.table is used in as.h2o and as.data.frame.H2OFrame only, are you suggesting separate property for each? I dont see any benefit in that, also I dont think we want to disable the use of data.table in as.h2o for automl, we want to use it if available

Reply:
Not a property, an option passed to `as.h2o` function. I think that would be beneficial to be able to bypass data.table on case-by-case basis.

>  I dont think we want to disable the use of data.table in as.h2o for automl, we want to use it if available

DT doesn't add any value in this particular case. The only reason why DT was added to H2O as an option is that it is faster on larger data. On small data (that we handle in this method) it doesn't matter.

Reply:
I added `use_datatable` option to `as.h2o` and removed the `suppressWarnings` from the automl's call so we don't suppress other potentially useful warnings.

Reply:
And I manually tested it again with these 3 scenarios:

- no data.table installed
- data.table but no bit64 installed
- data.table and bit64 installed